test/xspec-777.xspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<x:description query="my-namespace" query-at="xspec-777.xquery" stylesheet="xspec-777.xsl"
+	xmlns:my="my-namespace" xmlns:x="http://www.jenitennison.com/xslt/xspec">
+
+	<x:param as="document-node(element(test))" href="xspec-777.xml" name="my:test" />
+
+	<x:scenario>
+		<x:label>When x:param[@href] in XSpec defines $my:test and x:variable (or its XQuery
```

Comment:
Do you mean `xsl:variable` instead of `x:variable`?

Reply:
Fixed in f3dcdde310da818d355535c9cd8a1fdfddeff7bb